ci-operator/step-registry/ovn/conf/ipsec-user-mc/ovn-conf-ipsec-user-mc-commands.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
I think the best place would be to update `ovn-conf-ipsec-manifest-commands.sh` to rollout cluster with user defined IPsec machine configs and configure IPsec with it.
The `cp-ipi-ovn-ipsec-libreswan-f28` job may instruct `ovn-conf-ipsec-manifest` step to use user defined machine config via new env config variable.
https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-provision-chain.yaml

Reply:
As `ovn-conf-ipsec-manifest` is manifest configuration before the cluster installation, do you mean that I need to put the user defined IPsec machine configs to manifest files just as the way below `# additional os extension for 4.14 only` ?

Reply:
ok, nice, can see configs are applied in order now https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/66954/rehearse-66954-periodic-ci-openshift-openshift-tests-private-release-4.20-amd64-nightly-gcp-ipi-ovn-ipsec-libreswan-f28/1950393364230705152/artifacts/gcp-ipi-ovn-ipsec-libreswan-f28/ovn-conf-ipsec-manifest/build-log.txt.
running rehearse job for other ipsec lanes to confirm this.